src/Mesh/babylon.geometry.ts

```diff
@@ -239,6 +239,11 @@
             if (kind === VertexBuffer.PositionKind) {
                 if (totalVertices != null) {
                     this._totalVertices = totalVertices;
+                } else {
+                    var data = <FloatArray>buffer.getData();
+                    if (data != null) {
+                        this._totalVertices = data.length / (buffer.byteStride * 4);

```

Comment:
it is actually / 4 :)
I'll fix it

Reply:
Lol

Reply:
https://github.com/BabylonJS/Babylon.js/pull/4069